tests/postgres/select_test.go

```diff
@@ -2251,6 +2251,37 @@ FOR`
 	}
 }
 
+func TestRowLockWithJoins(t *testing.T) {
+	expectedSQL := `
+SELECT *
+FROM dvds.film
+     INNER JOIN dvds.film_category ON (film_category.film_id = film.film_id)
+     LEFT JOIN dvds.film_actor ON (film_actor.film_id = film.film_id)
+LIMIT 1
+FOR UPDATE OF "film", "film_category" NOWAIT;
+`
+	query := Film.
```

Comment:
It doesn't make sense to use both classical and jet select statement in the same query. Since there is a `FROM` clause `Film.` can be removed.